src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -233,13 +233,13 @@ private void createBranchSensitivityValue(LfSensitivityFactor<DcVariableType, Dc
                     sensiValue += contingencyElement.getAlphaForSensitivityValue() * contingencySensitivity;
```

Comment:
I do think that sens =0 is confusing here as there could be some cases where contingency has not occurred on one particular line but still there is sens =0 instead of that I would suggest using enum to make it clear that this is the contingency case which has affected this line

Reply:
As the sensitivity analysis through open-loadflow is used a lot by the community of pypowsybl users, `NaN` is much more adapted for them.